index.bs

```diff
@@ -2365,48 +2365,99 @@ Reviews and Review Responsibilities</h4>
+	</ul>
+
+<h5 id="horizontal-review">
+Horizontal Review</h5>
+
+	The objective of <dfn id=dfn-horizontal-review export>horizontal review</dfn> is
+	to make sure specifications are robust in terms of accessibility, internationalisation,
+	privacy, security, and technical architecture by the time they become a [=W3C Recommendation=]. 
+
+	Horizontal review <em class="rfc2119">should</em> be sought continuously as a
+	specification matures, so that accessibility, internationalisation, privacy,
+	security, and technical architecture are considered throughout. 
+
+	Before a proposed specification transitions onto the Recommendation Track, the
+	group responsible for the specification <em class="rfc2119">must</em> solicit a
+	horizontal review from each of the horizontal review groups (in many cases,
+	submitting a self-review to the reviewing group will satisfy this requirement).
+
+	The <dfn id=dfn-horizontal-review-groups export>Horizontal Review Groups</dfn> are:
+
+	<ul>
+		<li><a href="https://www.w3.org/WAI/APA/">Accessible Platform Architectures Working Group</a></li>
+		<li><a href="https://www.w3.org/International/core/Overview">Internationalisation Working Group</a></li>
+		<li><a href="https://www.w3.org/Privacy/">Privacy Interest Group</a></li>
+		<li><a href="https://www.w3.org/2001/tag/">Technical Architecture Group</a></li>
+	</ul>
+
+	When the [=First Public Working Draft=] of a specification is published, a notification
+	<em class="rfc2119">must</em> be sent to
+	<a href="https://lists.w3.org/Archives/Public/horizontal-review-announce/">horizontal-review-announce@w3.org</a>. 
+
+	Horizontal review <em class="rfc2119">must</em> be solicited from the Horizontal
+	Review groups at least 60 days before requesting transition to [=Candidate Recommendation=]. 
+
+	Horizontal Review of all new and significantly revised sections of a specification
+	<em class="rfc2119">should</em> also be sought from the Horizontal Review groups. 
+
+	When a Working Group requests horizontal review: 
+
+	<ul>
+		<li>The horizontal review group <em class="rfc2119">must</em> acknowledge
+		 	receipt of the request and indicate whether the review will be conducted</li>
+		<li>The horizontal review group <em class="rfc2119">should</em> review
+			the specification and raise relevant issues within the agreed timeframe</li>
+		<li>The requesting Working Group <em class="rfc2119">must</em> acknowledge
+			receipt of each issue, then give each issue due consideration before
+			providing an appropriate response</li>
+		<li>If the Working Group and the horizontal review group do not agree,
+			they <em class="rfc2119">should</em> make a determined effort to find
+			agreement</li>
+		<li>When the Working Group requests transition of a specification, evidence
+			of this process <em class="rfc2119">must</em> be provided and any issues
+			that remain unresolved <em class="rfc2119">must</em> be indicated</li>
```

Comment:
While these last 3 bullets are true for issues raised during HR, they're not limited to HR, and are also true for issues raised by anybody at any time. If this needs to be clarified, it should be clarified in the "Advancement on the Recommendation Track" section. But redundant requirements with slightly different phrasing are not good, so I'd remove theses 3 bullet points from here.

Reply:
Happy to remove them but I wasn't sure if you meant they should be moved to the general section, replace what's already in that section, or just be outright deleted?